### PR TITLE
Added support for HK region

### PR DESCRIPTION
**Pull request by trev142**
Added support for the HK region which seems to be used by us here in Australia

### CODE REVIEW

**Review by kairoaraujo — COMMENTED** [review]
Hi @trev142, is it working properly for this region?
Because I was having issues on UE region.

File: kodaksmarthome/__version__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 __title__ = "python-kodaksmarthome"
 __description__ = "Python Kodak SmartHome API"
 __url__ = "https://github.com/kairoaraujo/kodak_smarthome_api"
-__version__ = "0.1.1"
```
Comment (kairoaraujo) [inline kodaksmarthome/__version__.py:10]: @trev142 , Just put as 0.2.0 as this is not a bug fix and some enhancement.

**Review by kairoaraujo — CHANGES_REQUESTED** [review]

**Review by kairoaraujo — APPROVED** [review]

File: kodaksmarthome/constants.py [hunk 2]
```diff
@@ -18,7 +18,14 @@
         "URL_AUTH": "https://app-eu.kodaksmarthome.com/web/authenticate",
         "URL_DEVICES": "https://app-eu.kodaksmarthome.com/web/user/device",
         "URL_LOGOUT": "https://app-eu.kodaksmarthome.com/web/#/user/logout",
-    }
+        },
```
Comment (kairoaraujo) [inline kodaksmarthome/constants.py:21]: The tests for PEP8 failed, please fix the following requirements:

```
pep8 run-test: commands[0] | flake8 --ignore=F401,W503
72
./kodaksmarthome/constants.py:21:9: E123 closing bracket does not match indentation of opening bracket's line
73
./kodaksmarthome/constants.py:22:1: W191 indentation contains tabs
74
./kodaksmarthome/constants.py:22:1: E101 indentation contains mixed spaces and tabs
75
./kodaksmarthome/constants.py:22:2: E131 continuation line unaligned for hanging indent
76
./kodaksmarthome/constants.py:23:1: E101 indentation contains mixed spaces and tabs
77
./kodaksmarthome/constants.py:28:9: E123 closing bracket does not match indentation of opening bracket's line
78
```

**Review by kairoaraujo — CHANGES_REQUESTED** [review]

**Comment by raghavrithvik** [comment]
How do I get this updated to work with the HA component?

**Comment by kairoaraujo** [comment]
Closed due to PR #13

**Pull request closed without merging**